### PR TITLE
Travel Pay/Update static-pages BTSSS login widget

### PR DESCRIPTION
**Pull request by liztownd**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- _(Summarize the changes that have been made to the platform)_
Updated content on the existing BTSSS login widget for use on the [Get Reimbursed for Travel page](https://staging.va.gov/health-care/get-reimbursed-for-travel-pay/). 

Moving one line out of static content (sitewide team is aware) into the widget so we can change it as we iterate the product. Adding a link to view the status on VA.gov (although for now the user must still navigate to external site to file a new claim).

- _(What is the solution, why is this the solution)_
We needed to add an entry point for the travel pay status page.

- _(Which team do you work for, does your team own the maintenance of this component?)_
Travel Pay

- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_
End date hopefully by EOY, but possibly now mid-January.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
[department-of-veterans-affairs/va.gov-team#83284](https://github.com/department-of-veterans-affairs/va.gov-team/issues/83284) (Authenticated widget)
[department-of-veterans-affairs/va.gov-team#97872](https://github.com/department-of-veterans-affairs/va.gov-team/issues/97872) (Unauthenticated widget)

## Testing done

- _Describe what the old behavior was prior to the change_
Widget worked, but once logged in it only sent the user to the external BTSSS portal.

- _Describe the steps required to verify your changes are working as expected_
Widget should still exist on [this page](https://staging.va.gov/health-care/get-reimbursed-for-travel-pay/), but now once you log in there should be a link to the VA.gov travel claim status page as well as the external link for filing a claim.
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| unauth  |    Screenshot 2024-12-10 at 12 47 50 PM    |   Screenshot 2024-12-10 at 12 48 00 PM    |
| auth |     Screenshot 2024-12-10 at 12 49 24 PM   |    Screenshot 2024-12-10 at 12 47 30 PM   |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*
`/src/applications/static-pages/BTSSS-login`

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/static-pages/BTSSS-login/AuthContext/index.jsx [hunk 1]
```diff
@@ -1,12 +1,58 @@
...
+  );
+};
+
+// AuthContext.propTypes = {
+//   appEnabled: PropTypes.bool,
+// };
+
+// const mapStateToProps = store => ({
+//   appEnabled: toggleValues(store)[FEATURE_FLAG_NAMES.travelPayPowerSwitch],
+// });
```
Comment (kjduensing) [inline src/applications/static-pages/BTSSS-login/AuthContext/index.jsx:54]: remove?

File: src/applications/static-pages/BTSSS-login/AuthContext/index.jsx [hunk 2]
```diff
@@ -1,12 +1,58 @@
...
 
 export default AuthContext;
+
+// export default connect(mapStateToProps)(AuthContext);
```
Comment (kjduensing) [inline src/applications/static-pages/BTSSS-login/AuthContext/index.jsx:58]: remove?

**Review by kjduensing — DISMISSED** [review]
looks good

File: src/applications/static-pages/BTSSS-login/AuthContext/index.jsx (hunk 2 above)
Comment (liztownd) [reply src/applications/static-pages/BTSSS-login/AuthContext/index.jsx]: Oops. sorry. Removed. :D

**Review by kjduensing — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**